### ISSUE TITLE
8muses gives Error: Null when downloading large amount of images

### ISSUE THREAD

**Issue by DinnerBlasterX** [RipMeApp/ripme#110]
[NSFW]

    Ripme version: 1.5.14
    Java version (output of java -version): 1.7.0_55
    Operating system: Windows 10
    Exact URL: anything really large (ex: https://www.8muses.com/comix/album/taboolicious_xxx-comics)
    Please include any additional information about how to reproduce the problem: Attempt to download an album with a few hundred (or thousand) images and eventually the program will give the error.

Expected Behavior

Downloads all the albums in an 8muses artists page and puts them in the download directory.  Albums with multiple parts are downloaded to the same folder in the rip.

Actual Behavior

Says 'rip complete' then Error: 'null' if the amount of items being downloaded is too large, then deletes the folder that was created in the download directory.

I would assume this may be an API thing, or something to do with too many requests. It may also be based on the number of albums rather than number of images, but it's difficult to tell. 

Ex:  (https://www.8muses.com/comix/album/various-authors/sillygirl) DOES work, and it has nested albums (girly watch pt 1-4 are under separate groups)

But something with a similar layout like (https://www.8muses.com/comix/album/taboolicious_xxx-comics) DOES NOT work.  Note that the latter link has many more images.

Downloads work fine in the 0-400 image range on a few albums, but don't seem to work for the same amount of images in 20-30 albums (some of which are inside other directories).

Note that it is possible to save all the same images from an artist by queuing the albums individually.

There may also be something obvious that I'm missing here, I'm new to this program.

P.S. I'm blind and didn't see the new repo sorry

**Comment by rautamiekka** [comment]
Confirmed, it'll scan the albums quickly (wish DeviantArt was so fast) but nothing is downloaded and the folder doesn't exist:

```
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/turnabout
Downloading https://www.8muses.com/comix/picture/224645/taboolicious_xxx-comics/turnabout/1
Downloading https://www.8muses.com/comix/picture/224646/taboolicious_xxx-comics/turnabout/2
Downloading https://www.8muses.com/comix/picture/224647/taboolicious_xxx-comics/turnabout/3
Downloading https://www.8muses.com/comix/picture/224648/taboolicious_xxx-comics/turnabout/4
Downloading https://www.8muses.com/comix/picture/224649/taboolicious_xxx-comics/turnabout/5
Downloading https://www.8muses.com/comix/picture/224650/taboolicious_xxx-comics/turnabout/6
Downloading https://www.8muses.com/comix/picture/224651/taboolicious_xxx-comics/turnabout/7
Downloading https://www.8muses.com/comix/picture/224652/taboolicious_xxx-comics/turnabout/8
Downloading https://www.8muses.com/comix/picture/224653/taboolicious_xxx-comics/turnabout/9
Downloading https://www.8muses.com/comix/picture/224654/taboolicious_xxx-comics/turnabout/10
Downloading https://www.8muses.com/comix/picture/224655/taboolicious_xxx-comics/turnabout/11
Downloading https://www.8muses.com/comix/picture/224656/taboolicious_xxx-comics/turnabout/12
Downloading https://www.8muses.com/comix/picture/224657/taboolicious_xxx-comics/turnabout/13
Downloading https://www.8muses.com/comix/picture/224658/taboolicious_xxx-comics/turnabout/14
Downloading https://www.8muses.com/comix/picture/224659/taboolicious_xxx-comics/turnabout/15
Downloading https://www.8muses.com/comix/picture/224660/taboolicious_xxx-comics/turnabout/16
Downloading https://www.8muses.com/comix/picture/224661/taboolicious_xxx-comics/turnabout/17
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/the-patient-is-in
Downloading https://www.8muses.com/comix/picture/182905/taboolicious_xxx-comics/the-patient-is-in/1
Downloading https://www.8muses.com/comix/picture/182906/taboolicious_xxx-comics/the-patient-is-in/2
Downloading https://www.8muses.com/comix/picture/182907/taboolicious_xxx-comics/the-patient-is-in/3
Downloading https://www.8muses.com/comix/picture/182908/taboolicious_xxx-comics/the-patient-is-in/4
Downloading https://www.8muses.com/comix/picture/182909/taboolicious_xxx-comics/the-patient-is-in/5
Downloading https://www.8muses.com/comix/picture/182910/taboolicious_xxx-comics/the-patient-is-in/6
Downloading https://www.8muses.com/comix/picture/182911/taboolicious_xxx-comics/the-patient-is-in/7
Downloading https://www.8muses.com/comix/picture/182912/taboolicious_xxx-comics/the-patient-is-in/8
Downloading https://www.8muses.com/comix/picture/182913/taboolicious_xxx-comics/the-patient-is-in/9
Downloading https://www.8muses.com/comix/picture/182914/taboolicious_xxx-comics/the-patient-is-in/10
Downloading https://www.8muses.com/comix/picture/182915/taboolicious_xxx-comics/the-patient-is-in/11
Downloading https://www.8muses.com/comix/picture/182916/taboolicious_xxx-comics/the-patient-is-in/12
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/sib-secret
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/sib-secret/issue-2-code-red
Downloading https://www.8muses.com/comix/picture/313232/taboolicious_xxx-comics/sib-secret/issue-2-code-red/1
Downloading https://www.8muses.com/comix/picture/313233/taboolicious_xxx-comics/sib-secret/issue-2-code-red/2
Downloading https://www.8muses.com/comix/picture/313234/taboolicious_xxx-comics/sib-secret/issue-2-code-red/3
Downloading https://www.8muses.com/comix/picture/313235/taboolicious_xxx-comics/sib-secret/issue-2-code-red/4
Downloading https://www.8muses.com/comix/picture/313236/taboolicious_xxx-comics/sib-secret/issue-2-code-red/5
Downloading https://www.8muses.com/comix/picture/328839/taboolicious_xxx-comics/sib-secret/issue-2-code-red/6
Downloading https://www.8muses.com/comix/picture/462506/taboolicious_xxx-comics/sib-secret/issue-2-code-red/7
Downloading https://www.8muses.com/comix/picture/472933/taboolicious_xxx-comics/sib-secret/issue-2-code-red/8
Downloading https://www.8muses.com/comix/picture/472934/taboolicious_xxx-comics/sib-secret/issue-2-code-red/9
Downloading https://www.8muses.com/comix/picture/481215/taboolicious_xxx-comics/sib-secret/issue-2-code-red/10
Downloading https://www.8muses.com/comix/picture/481216/taboolicious_xxx-comics/sib-secret/issue-2-code-red/11
Downloading https://www.8muses.com/comix/picture/502573/taboolicious_xxx-comics/sib-secret/issue-2-code-red/12
Downloading https://www.8muses.com/comix/picture/502574/taboolicious_xxx-comics/sib-secret/issue-2-code-red/13
Downloading https://www.8muses.com/comix/picture/572845/taboolicious_xxx-comics/sib-secret/issue-2-code-red/14
Downloading https://www.8muses.com/comix/picture/572846/taboolicious_xxx-comics/sib-secret/issue-2-code-red/15
Downloading https://www.8muses.com/comix/picture/572847/taboolicious_xxx-comics/sib-secret/issue-2-code-red/16
Downloading https://www.8muses.com/comix/picture/572848/taboolicious_xxx-comics/sib-secret/issue-2-code-red/17
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/sib-secret/issue-1
Downloading https://www.8muses.com/comix/picture/313219/taboolicious_xxx-comics/sib-secret/issue-1/1
Downloading https://www.8muses.com/comix/picture/313220/taboolicious_xxx-comics/sib-secret/issue-1/2
Downloading https://www.8muses.com/comix/picture/313221/taboolicious_xxx-comics/sib-secret/issue-1/3
Downloading https://www.8muses.com/comix/picture/313222/taboolicious_xxx-comics/sib-secret/issue-1/4
Downloading https://www.8muses.com/comix/picture/313223/taboolicious_xxx-comics/sib-secret/issue-1/5
Downloading https://www.8muses.com/comix/picture/313224/taboolicious_xxx-comics/sib-secret/issue-1/6
Downloading https://www.8muses.com/comix/picture/313225/taboolicious_xxx-comics/sib-secret/issue-1/7
Downloading https://www.8muses.com/comix/picture/313226/taboolicious_xxx-comics/sib-secret/issue-1/8
Downloading https://www.8muses.com/comix/picture/313227/taboolicious_xxx-comics/sib-secret/issue-1/9
Downloading https://www.8muses.com/comix/picture/313228/taboolicious_xxx-comics/sib-secret/issue-1/10
Downloading https://www.8muses.com/comix/picture/313229/taboolicious_xxx-comics/sib-secret/issue-1/11
Downloading https://www.8muses.com/comix/picture/313230/taboolicious_xxx-comics/sib-secret/issue-1/12
Downloading https://www.8muses.com/comix/picture/313231/taboolicious_xxx-comics/sib-secret/issue-1/13
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/over-time
Downloading https://www.8muses.com/comix/picture/182880/taboolicious_xxx-comics/over-time/1
Downloading https://www.8muses.com/comix/picture/182881/taboolicious_xxx-comics/over-time/2
Downloading https://www.8muses.com/comix/picture/182882/taboolicious_xxx-comics/over-time/3
Downloading https://www.8muses.com/comix/picture/182883/taboolicious_xxx-comics/over-time/4
Downloading https://www.8muses.com/comix/picture/182884/taboolicious_xxx-comics/over-time/5
Downloading https://www.8muses.com/comix/picture/182885/taboolicious_xxx-comics/over-time/6
Downloading https://www.8muses.com/comix/picture/182886/taboolicious_xxx-comics/over-time/7
Downloading https://www.8muses.com/comix/picture/182887/taboolicious_xxx-comics/over-time/8
Downloading https://www.8muses.com/comix/picture/182888/taboolicious_xxx-comics/over-time/9
Downloading https://www.8muses.com/comix/picture/182889/taboolicious_xxx-comics/over-time/10
Downloading https://www.8muses.com/comix/picture/182890/taboolicious_xxx-comics/over-time/11
Downloading https://www.8muses.com/comix/picture/182891/taboolicious_xxx-comics/over-time/12
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/one-hard-lesson
Downloading https://www.8muses.com/comix/picture/182871/taboolicious_xxx-comics/one-hard-lesson/1
Downloading https://www.8muses.com/comix/picture/182872/taboolicious_xxx-comics/one-hard-lesson/2
Downloading https://www.8muses.com/comix/picture/182873/taboolicious_xxx-comics/one-hard-lesson/3
Downloading https://www.8muses.com/comix/picture/182874/taboolicious_xxx-comics/one-hard-lesson/4
Downloading https://www.8muses.com/comix/picture/182875/taboolicious_xxx-comics/one-hard-lesson/5
Downloading https://www.8muses.com/comix/picture/182876/taboolicious_xxx-comics/one-hard-lesson/6
Downloading https://www.8muses.com/comix/picture/182877/taboolicious_xxx-comics/one-hard-lesson/7
Downloading https://www.8muses.com/comix/picture/182878/taboolicious_xxx-comics/one-hard-lesson/8
Downloading https://www.8muses.com/comix/picture/182879/taboolicious_xxx-comics/one-hard-lesson/9
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/math-is-fun
Downloading https://www.8muses.com/comix/picture/254186/taboolicious_xxx-comics/math-is-fun/1
Downloading https://www.8muses.com/comix/picture/254187/taboolicious_xxx-comics/math-is-fun/2
Downloading https://www.8muses.com/comix/picture/254188/taboolicious_xxx-comics/math-is-fun/3
Downloading https://www.8muses.com/comix/picture/254189/taboolicious_xxx-comics/math-is-fun/4
Downloading https://www.8muses.com/comix/picture/254190/taboolicious_xxx-comics/math-is-fun/5
Downloading https://www.8muses.com/comix/picture/254191/taboolicious_xxx-comics/math-is-fun/6
Downloading https://www.8muses.com/comix/picture/254192/taboolicious_xxx-comics/math-is-fun/7
Downloading https://www.8muses.com/comix/picture/254193/taboolicious_xxx-comics/math-is-fun/8
Downloading https://www.8muses.com/comix/picture/254194/taboolicious_xxx-comics/math-is-fun/9
Downloading https://www.8muses.com/comix/picture/254195/taboolicious_xxx-comics/math-is-fun/10
Downloading https://www.8muses.com/comix/picture/254196/taboolicious_xxx-comics/math-is-fun/11
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/happy-accident
Downloading https://www.8muses.com/comix/picture/254177/taboolicious_xxx-comics/happy-accident/1
Downloading https://www.8muses.com/comix/picture/254178/taboolicious_xxx-comics/happy-accident/2
Downloading https://www.8muses.com/comix/picture/254179/taboolicious_xxx-comics/happy-accident/3
Downloading https://www.8muses.com/comix/picture/254180/taboolicious_xxx-comics/happy-accident/4
Downloading https://www.8muses.com/comix/picture/254181/taboolicious_xxx-comics/happy-accident/5
Downloading https://www.8muses.com/comix/picture/254182/taboolicious_xxx-comics/happy-accident/6
Downloading https://www.8muses.com/comix/picture/254183/taboolicious_xxx-comics/happy-accident/7
Downloading https://www.8muses.com/comix/picture/254184/taboolicious_xxx-comics/happy-accident/8
Downloading https://www.8muses.com/comix/picture/254185/taboolicious_xxx-comics/happy-accident/9
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/halloween-2016
Downloading https://www.8muses.com/comix/picture/511428/taboolicious_xxx-comics/halloween-2016/1
Downloading https://www.8muses.com/comix/picture/514855/taboolicious_xxx-comics/halloween-2016/2
Downloading https://www.8muses.com/comix/picture/514856/taboolicious_xxx-comics/halloween-2016/3
Downloading https://www.8muses.com/comix/picture/511429/taboolicious_xxx-comics/halloween-2016/4
Downloading https://www.8muses.com/comix/picture/511430/taboolicious_xxx-comics/halloween-2016/5
Downloading https://www.8muses.com/comix/picture/511431/taboolicious_xxx-comics/halloween-2016/6
Downloading https://www.8muses.com/comix/picture/514857/taboolicious_xxx-comics/halloween-2016/7
Downloading https://www.8muses.com/comix/picture/526195/taboolicious_xxx-comics/halloween-2016/8
Downloading https://www.8muses.com/comix/picture/526196/taboolicious_xxx-comics/halloween-2016/9
Downloading https://www.8muses.com/comix/picture/526197/taboolicious_xxx-comics/halloween-2016/10
Downloading https://www.8muses.com/comix/picture/526198/taboolicious_xxx-comics/halloween-2016/11
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/halloween-2015
Downloading https://www.8muses.com/comix/picture/254172/taboolicious_xxx-comics/halloween-2015/1
Downloading https://www.8muses.com/comix/picture/254173/taboolicious_xxx-comics/halloween-2015/2
Downloading https://www.8muses.com/comix/picture/254174/taboolicious_xxx-comics/halloween-2015/3
Downloading https://www.8muses.com/comix/picture/254175/taboolicious_xxx-comics/halloween-2015/4
Downloading https://www.8muses.com/comix/picture/254176/taboolicious_xxx-comics/halloween-2015/5
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/family-values
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/family-values/best-weekend-ever
Downloading https://www.8muses.com/comix/picture/182853/taboolicious_xxx-comics/family-values/best-weekend-ever/1
Downloading https://www.8muses.com/comix/picture/182854/taboolicious_xxx-comics/family-values/best-weekend-ever/2
Downloading https://www.8muses.com/comix/picture/182855/taboolicious_xxx-comics/family-values/best-weekend-ever/3
Downloading https://www.8muses.com/comix/picture/182856/taboolicious_xxx-comics/family-values/best-weekend-ever/4
Downloading https://www.8muses.com/comix/picture/182857/taboolicious_xxx-comics/family-values/best-weekend-ever/5
Downloading https://www.8muses.com/comix/picture/182858/taboolicious_xxx-comics/family-values/best-weekend-ever/6
Downloading https://www.8muses.com/comix/picture/182859/taboolicious_xxx-comics/family-values/best-weekend-ever/7
Downloading https://www.8muses.com/comix/picture/182860/taboolicious_xxx-comics/family-values/best-weekend-ever/8
Downloading https://www.8muses.com/comix/picture/182861/taboolicious_xxx-comics/family-values/best-weekend-ever/9
Downloading https://www.8muses.com/comix/picture/182862/taboolicious_xxx-comics/family-values/best-weekend-ever/10
Downloading https://www.8muses.com/comix/picture/182863/taboolicious_xxx-comics/family-values/best-weekend-ever/11
Downloading https://www.8muses.com/comix/picture/182864/taboolicious_xxx-comics/family-values/best-weekend-ever/12
Downloading https://www.8muses.com/comix/picture/182865/taboolicious_xxx-comics/family-values/best-weekend-ever/13
Downloading https://www.8muses.com/comix/picture/182866/taboolicious_xxx-comics/family-values/best-weekend-ever/14
Downloading https://www.8muses.com/comix/picture/182867/taboolicious_xxx-comics/family-values/best-weekend-ever/15
Downloading https://www.8muses.com/comix/picture/182868/taboolicious_xxx-comics/family-values/best-weekend-ever/16
Downloading https://www.8muses.com/comix/picture/182869/taboolicious_xxx-comics/family-values/best-weekend-ever/17
Downloading https://www.8muses.com/comix/picture/182870/taboolicious_xxx-comics/family-values/best-weekend-ever/18
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/easter-heat-2016
Downloading https://www.8muses.com/comix/picture/323814/taboolicious_xxx-comics/easter-heat-2016/1
Downloading https://www.8muses.com/comix/picture/323815/taboolicious_xxx-comics/easter-heat-2016/2
Downloading https://www.8muses.com/comix/picture/323816/taboolicious_xxx-comics/easter-heat-2016/3
Downloading https://www.8muses.com/comix/picture/323817/taboolicious_xxx-comics/easter-heat-2016/4
Downloading https://www.8muses.com/comix/picture/323818/taboolicious_xxx-comics/easter-heat-2016/5
Downloading https://www.8muses.com/comix/picture/323819/taboolicious_xxx-comics/easter-heat-2016/6
Downloading https://www.8muses.com/comix/picture/323820/taboolicious_xxx-comics/easter-heat-2016/7
Downloading https://www.8muses.com/comix/picture/327458/taboolicious_xxx-comics/easter-heat-2016/8
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/easter-2015
Downloading https://www.8muses.com/comix/picture/254165/taboolicious_xxx-comics/easter-2015/1
Downloading https://www.8muses.com/comix/picture/254166/taboolicious_xxx-comics/easter-2015/2
Downloading https://www.8muses.com/comix/picture/254167/taboolicious_xxx-comics/easter-2015/3
Downloading https://www.8muses.com/comix/picture/254168/taboolicious_xxx-comics/easter-2015/4
Downloading https://www.8muses.com/comix/picture/254169/taboolicious_xxx-comics/easter-2015/5
Downloading https://www.8muses.com/comix/picture/254170/taboolicious_xxx-comics/easter-2015/6
Downloading https://www.8muses.com/comix/picture/254171/taboolicious_xxx-comics/easter-2015/7
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/blind-date
Downloading https://www.8muses.com/comix/picture/264801/taboolicious_xxx-comics/blind-date/1
Downloading https://www.8muses.com/comix/picture/264802/taboolicious_xxx-comics/blind-date/2
Downloading https://www.8muses.com/comix/picture/264803/taboolicious_xxx-comics/blind-date/3
Downloading https://www.8muses.com/comix/picture/264804/taboolicious_xxx-comics/blind-date/4
Downloading https://www.8muses.com/comix/picture/264805/taboolicious_xxx-comics/blind-date/5
Downloading https://www.8muses.com/comix/picture/264806/taboolicious_xxx-comics/blind-date/6
Downloading https://www.8muses.com/comix/picture/264807/taboolicious_xxx-comics/blind-date/7
Downloading https://www.8muses.com/comix/picture/264808/taboolicious_xxx-comics/blind-date/8
Downloading https://www.8muses.com/comix/picture/264809/taboolicious_xxx-comics/blind-date/9
Downloading https://www.8muses.com/comix/picture/264810/taboolicious_xxx-comics/blind-date/10
Downloading https://www.8muses.com/comix/picture/264811/taboolicious_xxx-comics/blind-date/11
Downloading https://www.8muses.com/comix/picture/264812/taboolicious_xxx-comics/blind-date/12
Downloading https://www.8muses.com/comix/picture/264813/taboolicious_xxx-comics/blind-date/13
Downloading https://www.8muses.com/comix/picture/264814/taboolicious_xxx-comics/blind-date/14
Downloading https://www.8muses.com/comix/picture/264815/taboolicious_xxx-comics/blind-date/15
Downloading https://www.8muses.com/comix/picture/278183/taboolicious_xxx-comics/blind-date/16
Downloading https://www.8muses.com/comix/picture/279769/taboolicious_xxx-comics/blind-date/17
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/art
Downloading https://www.8muses.com/comix/album/taboolicious_xxx-comics/art/2011
Rip complete, saved to G:\_DOWNLOADS_\RipMe\rips\8muses_taboolicious_xxx-comics
null
```

Window$ 7 Ultimate SP1 x64
Oracle Java SE 8 Update 144 x64 (installing 152 now)
RipMe 1.5.14

**Comment by DinnerBlasterX** [comment]
Good to know!  If you need me to do anything I'm around to test, otherwise hopefully you guys can figure it out :)

**Comment by cyian-1756** [comment]
After a bit of debugging it seems that getFullSizeImage() is not returning the full size image url, but rather null. It's probably just another css change and I should have it fixed today

**Comment by cyian-1756** [comment]
This should be fixed in 1.5.15
